### DISCUSSION TITLE
Community Meeting 2024-06-17

### DISCUSSION THREAD

**Discussion by dustinblack** [arcalot/.github#15] [Community]
# Agenda

- [x] https://github.com/arcalot/arcalot-round-table/pull/42
- [x] https://github.com/arcalot/arcalot-round-table/pull/45
- [x] https://github.com/arcalot/arcalot-round-table/pull/48
- [x] https://github.com/arcalot/arcaflow-engine/pull/182
- [x] CNCF go/no-go
- [x] ART inactivity review https://github.com/arcalot/arcalot-round-table/pull/50
- [x] https://github.com/arcalot/arcaflow-engine-deployer-docker/issues/1

**Comment by webbnh** [comment]
- Arcaflow's use of Python:  should it be uniformly on 3.9 (or 3.12), with support for 3.9-3.12?
- The [`arcaflow-plugin-utilities` update](https://github.com/arcalot/arcaflow-plugin-utilities/pull/40) of `quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase` (and `arcaflow-plugin-baseimage-python-osbase`) from `0.3.1` to `0.4.0`, which is a change from CentOS Stream 8 to Stream 9.

**Comment by webbnh** [comment]
- Managing old container images:  e.g., `quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.2.0` contains "critical" CVEs, and I assume that we are not going to address this by rebuilding that image...so...should we delete it?

**Comment by webbnh** [comment]
- The Renovate bot's schedule:  for some repo's, it is set to create new PRs only on Sunday-night/Monday-morning; for some, it can create them at any time...it seems odd that we're not consistent on this.

**Comment by dustinblack** [comment]
I propose we do the ART inactivity review as agenda item #1. We have open proposals that have not met the voting quorum based on current ART membership, but I believe some of those members should already be auto-removed from the ART according to the charter.

**Comment by dustinblack** [comment]
TODO: Update getting started guide at arcalot.io based on feedback from @Harshith-umesh

**Reply by Harshith-umesh** [reply]
https://docs.google.com/document/d/1VVmGX7PPSJn8WMbG7mZ_XME1b6REI0LOCgOh8Ov_luY/edit?usp=sharing
